.experimenter.json

```diff
@@ -18,14 +18,25 @@
       "message-location": {
         "description": "Where is the message to be put.",
         "enum": [
+          "app-menu-item",
```

Comment:
accidental commit?

Reply:
this `.experimenter.json` stuff is killing me too, lol

Reply:
Sorry about that XD

Reply:
Doesn't matter if we commit this or not. The ordering is non-deterministic so it will keep flipping regardless. 😂